### ISSUE TITLE
navbar-right gets misaligned when using navbar-fixed-top

### ISSUE THREAD

**Issue by vanillajonathan** [twbs/bootstrap#16183]
Notice how the `navbar-right` text gets misaligned and placed too much right when using `navbar-fixed-top`.

http://jsbin.com/qesisekija/1/

**Comment by twbs-lmvtfy** [comment]
Hi @vanillajonathan!

You appear to have posted a live example (http://jsbin.com/jebuxaquhe/1/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 50, column 80: Duplicate ID ```` bs-example-navbar-collapse-5 ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by vanillajonathan** [comment]
Updated.
http://jsbin.com/qesisekija/1/

**Comment by benjokho** [comment]
The same issue appears when using `navbar-fixed-bottom`.

http://jsbin.com/bopaledadu/1/

**Comment by twbs-lmvtfy** [comment]
Hi @benjokho!

You appear to have posted a live example (http://jsbin.com/bopaledadu/1/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 7, column 24: ```` " ```` in an unquoted attribute value. Probable causes: Attributes running together or a URL query string in an unquoted attribute value.
* line 7, column 30: ```` " ```` in an unquoted attribute value. Probable causes: Attributes running together or a URL query string in an unquoted attribute value.
* line 7, column 31: Internal encoding declaration named an unsupported chararacter encoding ```` a"utf-8" ````.
* line 7, column 31: Bad value ```` a"utf-8" ```` for attribute ```` charset ```` on element ```` meta ````: Value contained ```` " ````, which is not a valid character in an encoding name.
* line 75, column 80: Duplicate ID ```` bs-example-navbar-collapse-2 ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by benjokho** [comment]
Updated.
http://jsbin.com/kayuvohumi/1/

**Comment by cvrebert** [comment]
Duplicate of #15706.
